sdks/java/io/hcatalog/src/main/java/org/apache/beam/sdk/io/hcatalog/HCatalogIO.java

```diff
@@ -186,6 +216,26 @@ public Read withFilter(String filter) {
       return toBuilder().setFilter(filter).build();
     }
 
+    /** Sets the duration after which the PartitionPollerFn resumes polling for new partitions. */
+    public Read withPollingInterval(Duration pollingInterval) {
+      return toBuilder().setPollingInterval(pollingInterval).build();
+    }
+
+    /** Specifies the name of the column which contains the watermark values. */
```

Comment:
This comment looks wrong. I'm generally not sure what this property is for.

Reply:
Removed. Older property. Not needed.